aspnetcore/mvc/views/razor.md

```diff
@@ -92,6 +100,20 @@ You can use an explicit expression to concatenate text with an expression result
 
 Without the explicit expression, `<p>Age@joe.Age</p>` is treated as an email address, and `<p>Age@joe.Age</p>` is rendered. When written as an explicit expression, `<p>Age33</p>` is rendered.
 
+You can also use explicit expressions to render output from generic methods in _.cshtml_ files.  In an implicit expression, the characters inside the brackets (`<>`) would be interpreted as an HTML tag:
```

Comment:
* \_.cshtml\_ :point_right: \*.cshtml\*
* Remove one space: "files.  In"
* would be :point_right: is

Reply:
I think you mean "are" not "is" for that last

Reply:
oh ... lol ... yes!

Reply:
so much for trying to work on three things **at once** :smile: